### DISCUSSION TITLE
that would be great if there is an option to wrap long lines

### DISCUSSION THREAD

**Discussion by RedXBeard** [BigTows/PinNote#29] [Ideas]
well can be done without wrapping but with sub category or whatever its called but i prefer wrapping if  it is possible? is there any way to get the size of the box?

**Comment by BigTows** [comment]
Hello, you talk about text of tasks?
If you can present screenshot of uncomfortable situation.

**Reply by RedXBeard** [reply]
Hi, 
This shot would be good to explain what i wanted to say, right?

<img width="603" alt="Screen Shot 2021-04-20 at 16 45 00" src="https://user-images.githubusercontent.com/7275194/115406509-ee2f3f80-a1f7-11eb-851f-0d337c08be87.png">

**Reply by BigTows** [reply]
Ohh, okay

I'll try to think of something

Sorry for big timeout response, didn't  see notification from GitHub.
